Tests/integration/Radius.Tests.ps1

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright 2019, Alexis La Goutte <alexis dot lagoutte at gmail dot com>
+# Copyright 2019, Cédric Moreau <moreaucedric0 at gmail dot com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+. ../common.ps1
+
+Describe  "Get-ArubaSWRadius" {
+    It "Get-ArubaSWRadius Does not throw an error" {
+        { Get-ArubaSWRadius } | Should Not Throw
+    }
+}
+
+Describe  "Add-ArubaSWRadius" {
+
+    It "Add-ArubaSWRadius mandatory parameters" {
+        Add-ArubaSWRadius -address 192.0.2.1 -shared_secret powerarubasw
+        $radius = Get-ArubaSWRadius -id 1
```

Comment:
filter with ip address ? (not sur it will be always id 1 if there is multiple radius server entry...)